src/vs/workbench/contrib/files/browser/views/explorerViewer.ts

```diff
@@ -486,19 +521,46 @@ export class FilesFilter implements ITreeFilter<ExplorerItem, FuzzyScore> {
 	}
 
 	updateConfiguration(): boolean {
+		interface IWorkingSetsConfiguration {
+			'working-sets'?: {
+				'working-sets': {
+					[key: string]: string[]
+				}
+			};
```

Comment:
Hey! I'm kinda new to TypeScript, just poking around, trying to learn some stuff 😄 . I've never seen this type of Interface before. Why don't you define `IWorkingSetsConfiguration` like this instead:

`'working-sets'?:{
   [key:string]: string[]
}`

Never seen this pattern before, just curious 😄